ogr/ogrsf_frmts/filegdb/FGdbDriver.cpp

```diff
@@ -847,6 +847,11 @@ void RegisterOGRFileGDB()
     poDriver->SetMetadataItem( GDAL_DMD_LONGNAME,
                                 "ESRI FileGDB" );
     poDriver->SetMetadataItem( GDAL_DCAP_VECTOR, "YES");
+    poDriver->SetMetadataItem( GDAL_DCAP_DELETE_LAYER, "YES" );
+    poDriver->SetMetadataItem( GDAL_DCAP_CREATE_LAYER, "YES" );
+    poDriver->SetMetadataItem( GDAL_DCAP_CREATE_FIELD, "YES" );
+    poDriver->SetMetadataItem( GDAL_DCAP_DELETE_FIELD, "YES" );
+    poDriver->SetMetadataItem( GDAL_DCAP_REORDER_FIELDS, "YES" ); // although marked as "TBD" in FGdbLayer.cpp!
```

Comment:
this line should be removed. And the 2 lines in FGdbLayer that advertize OLCReorderFields should be removed, as they are definitely not appropriate

Reply:
done